common/common-client/src/main/java/org/ow2/proactive/authentication/FileLoginModule.java

```diff
@@ -207,22 +219,37 @@ protected boolean logUser(String username, String password, boolean printErrorMe
      * @throws LoginException if login file is not found or unreadable.
      */
     private boolean authenticateUserFromFile(String username, String password) throws LoginException {
+
+
         Properties props = new Properties();
+        PrivateKey privateKey = null;
         try {
+            privateKey = getPrivateKey();
             FileInputStream stream = new FileInputStream(new File(loginFile));
             props.load(stream);
             stream.close();
+        } catch (KeyException e) {
+            throw new LoginException(e.toString());
         } catch (FileNotFoundException e) {
             throw new LoginException(e.toString());
         } catch (IOException e) {
             throw new LoginException(e.toString());
         }
 
         // verify the username and password
-        if (!props.containsKey(username) || !props.get(username).equals(password)) {
+        if (!props.containsKey(username)) {
             return false;
-        } else
+        } else {
+            String encryptedPassword = (String) props.get(username);
```

Comment:
yes, the login file is read as a Properties file. Thus it contains login name as keys and the values are the encrypted passwords